guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -3,11 +3,14 @@
 
 Use this procedure if your {ProjectServer} is not connected to the Red{nbsp}Hat Content Delivery Network.
 
+// TODO reuse common/modules/snip_using_installer_noop.adoc
```

Comment:
I don't see a solution for this unless bullet points could be removed from the warning but each bullet point deals with a separate topic so having them is logical.

Reply:
Agreed. I'm thinking of rephrasing it to something like "keep in sync with".